test/sim/sim_test_dae.cpp

```diff
@@ -72,8 +72,8 @@ double sim_solver_tolerance_dae(std::string const& inString)
 
 double sim_solver_tolerance_algebraic_dae(std::string const& inString)
 {
-    if (inString == "IRK")  return 1e-4;
-    if (inString == "GNSF") return 1e-4;
+    if (inString == "IRK")  return 3e-4;
+    if (inString == "GNSF") return 3e-4;
```

Comment:
why?

Reply:
I changed the example used in the test, adding more nonlinearities to the algebraic equations. Thus, I had to adjust the accuracy required to pass the test.

Reply:
Ok! Is it not possible to change the integrator options to give higher accuracy solutions? Even 1e-4 seems a bit high for a test.

Reply:
Not really.
The output for algebraic variables gets computed without performing al Newton-iteration, but by evaluating the polynomial through the values of the first stage.
Other things could be implemented, but I think are not priority...